src/pages/apprentissage-entreprises/index.page.test.tsx

```diff
@@ -29,6 +29,22 @@ jest.mock('~/server/start', () => ({
 }));
 
 describe('<ApprentissageEntreprises />', () => {
+	it('doit rendre du HTML respectant la specification', () => {
+		mockSmallScreen();
+		mockUseRouter({ query: { page: '1' } });
```

Comment:
```suggestion
		mockUseRouter();
```
D'ailleurs ça a l'air d'être pareil sur tous les tests du fichier

Reply:
On pouvait même le virer d'ailleurs